CONTRIBUTING.md

```diff
@@ -20,23 +19,18 @@ This repository store the sources for the website.
 To add a new event, you will need to manually create an event in the folder `_events`. The metadata for events is a bit more complex, it looks like:
 
 ---
-tags: [talk, training]
-title: "Test of DNA extraction and sequencing"
-starts: 2019-11-10
-ends: 2019-11-10
-location:
-  city: Freiburg
-  region: Baden Würtemberg
-  country: Germany
-supporters:
+
+tags: [talk, training] title: "Test of DNA extraction and sequencing" starts: 2019-11-10 ends: 2019-11-10 location: city: Freiburg region: Baden Würtemberg country: Germany supporters:
```

Comment:
Any reason to remove the line breaks?

Reply:
no, that's an issue... thanks for catching it, will fix that